external/src/test/groovy/org/hawkular/alerts/rest/ActionsITest.groovy

```diff
@@ -283,6 +284,7 @@ class ActionsITest extends AbstractQuarkusITestBase {
     }
 
     @Test
+    @Disabled("The actions store is no longer used by the engine. This is still an experimental change so let's keep this test around for a while.")
```

Comment:
I disabled this test and the other one below because they won't succeed unless the actions are persisted in Infinispan.